tests/query/test_query.py

```diff
@@ -54,3 +55,46 @@ def test_group_by_noisy_typed_order_desc(self):
         private_reader = PrivateQuery(reader, schema, 1.0)
         rs = private_reader.execute_typed("SELECT COUNT(*) AS c, married AS m FROM PUMS.PUMS GROUP BY married ORDER BY c DESC")
         assert(rs['c'][0] > rs['c'][1])
+    def test_no_tau(self):
+        # should never drop rows
+        reader = CSVReader(schema, df)
+        private_reader = PrivateQuery(reader, schema, 4.0)
+        for i in range(10):
+            rs = private_reader.execute_typed("SELECT COUNT(*) AS c FROM PUMS.PUMS WHERE age > 90 AND educ = '8'")
+            assert(len(rs['c']) == 1)
+    def test_no_tau_noisy(self):
```

Comment:
There can be some code sharing here but for now we can leave as is.

I can look into fixtures and code sharing for similar queries and tests.

Also, it is good to avoid prints during tests since they can add up to a noisy log file. logging.getLogger(__name__).debug("message") helps with this since it can be configured to show or not